src/main/java/checkout/CheckoutService.java

```diff
@@ -22,130 +21,17 @@ public CheckoutService(final Properties prop) {
-        PaymentsDetailsResponse paymentsDetailsResponse = checkout.paymentsDetails(paymentsDetailsRequest);
-        System.out.println("paymentsDetails response:\n" + paymentsDetailsResponse.toString());
-        return paymentsDetailsResponse;
-    }
-
-
-    private Amount getAmount(String type) {
-
-        String currency;
-
-        switch (type) {
-            case "alipay":
-                currency = "CNY";
-                break;
-            case "dotpay":
-                currency = "PLN";
-                break;
-            case "boletobancario":
-                currency = "BRL";
-                break;
-            case "ach":
-            case "paypal":
-                currency = "USD";
-                break;
-            default:
-                currency = "EUR";
-        }
-        Amount amount = new Amount();
-
-        amount.setCurrency(currency);
-        amount.setValue(1000L);
-        return amount;
-    }
-
-    private static void addLineItems(PaymentsRequest paymentsRequest) {
-        String item1 = "{\n" +
-                "                \"quantity\": \"1\",\n" +
-                "                \"amountExcludingTax\": \"450\",\n" +
-                "                \"taxPercentage\": \"1111\",\n" +
-                "                \"description\": \"Sunglasses\",\n" +
-                "                \"id\": \"Item #1\",\n" +
-                "                \"taxAmount\": \"50\",\n" +
-                "                \"amountIncludingTax\": \"500\",\n" +
-                "                \"taxCategory\": \"High\"\n" +
-                "            }";
-        String item2 = "{\n" +
-                "                \"quantity\": \"1\",\n" +
-                "                \"amountExcludingTax\": \"450\",\n" +
-                "                \"taxPercentage\": \"1111\",\n" +
-                "                \"description\": \"Headphones\",\n" +
-                "                \"id\": \"Item #2\",\n" +
-                "                \"taxAmount\": \"50\",\n" +
-                "                \"amountIncludingTax\": \"500\",\n" +
-                "                \"taxCategory\": \"High\"\n" +
-                "            }";
-
-        Gson gson = new GsonBuilder().create();
-        LineItem lineItem1 = gson.fromJson(item1, LineItem.class);
-        LineItem lineItem2 = gson.fromJson(item2, LineItem.class);
-
-        paymentsRequest.addLineItemsItem(lineItem1);
-        paymentsRequest.addLineItemsItem(lineItem2);
+        Amount amount = new Amount()
+                .currency("EUR")
+                .value(1000L);
+        CreateCheckoutSessionRequest checkoutSession = new CreateCheckoutSessionRequest();
+        checkoutSession.merchantAccount(merchantAccount);
+        checkoutSession.setChannel(CreateCheckoutSessionRequest.ChannelEnum.WEB);
+        checkoutSession.setReference(orderRef); // required
```

Comment:
The rest is required as well, I don't think the comment brings much information?  Or maybbe add some extra information when we create the orderRef ("typically generated by the customer CMS" for example).